dask/dataframe/core.py

```diff
@@ -4123,6 +4123,12 @@ def to_timedelta(arg, unit='ns', errors='raise'):
                           meta=meta)
 
 
+if hasattr(pd, isna):
```

Comment:
This should be `if hasattr(pd, 'isna'):` (as seen in the travis logs).

---

In general, I recommend trying to run the tests yourself before pushing to github. We use `py.test` for testing:

```bash
$ conda install pytest
# or
$ pip install pytest

$ py.test dask  # run the whole test suite
$ py.test dask/dataframe/tests/test_dataframe.py -k test_isna  # test just your added test
```
```

Reply:
Thanks! Should be fixed now.